lblod/ember-rdfa-editor-lblod-plugins
# Feat/redesign variable pills

### Overview
Redesign variable pills

##### connected issues and PRs:
GN-5121
https://github.com/lblod/ember-rdfa-editor/pull/1225


### Setup
None

### How to test/reproduce
Insert the different types of variables and check that they are in line with the design, also when you remove all the content from a variable the placeholder should reappear

### Challenges/uncertainties
None



### Checks PR readiness
